docs/setup.md

```diff
@@ -52,6 +52,7 @@ Minimum XCUITest driver version | Minimum required Xcode version
 3.32.0 | Xcode 10.2
 3.56.0 | Xcode 11
 4.2.0 | Xcode 12
+5.0.0 | Xcode 13
```

Comment:
Oh, here should be 4.8.0

Reply:
let me update here to update the public docs

Reply:
https://appium.github.io/appium-xcuitest-driver/4.28/setup/#xcode-version-support